### PR TITLE
Add additional test for waf + container logs & check null fields

### PR DESCRIPTION
**Pull request by jason-linthwaite**
I had a problem with the logs I was receiving from ALB. Potentially due to configuration, but there were an extra two fields (undocumented on this page: https://docs.aws.amazon.com/elasticloadbalancing/latest/application/load-balancer-access-logs.html) of "-" and "-" at the end of the log (see test).

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -56,6 +56,10 @@ function parseAlbLogLine(line) {
     if (finished) {
       if (element) {
         const fieldName = fields[counter]
+
+        // Allow for extra fields in the ALB log
+        if (typeof fieldName === 'undefined' || fieldName == null) continue;
```
Comment (PopeFelix) [inline index.js:61]: Oh, *that's* how you do that! I learned a thing. :)

**Comment by igtm** [comment]
We had a similar PR to this, and your idea is referred [here](https://github.com/igtm/node-alb-log-parser/pull/5/files#diff-e727e4bdf3657fd1d798edcd6b099d6e092f8573cba266154583a746bba0f346R68-R71).
so I pulled that PR and this issue is resolved.
Thank you so much for your contribution!
